### PR TITLE
Setup hello world

### PR DESCRIPTION
**Pull request by adel-gu**
## In this PR I made the following changes:

- Create a rails app by running the command `rails new -j webpack react-rails-helloworld`
- Install needed dependencies such as react, redux, and babel.
- Configure the file ` babel.config.js`.
- Generate and set up the Root controller, view, and route.
- Update webpack.config.js to include the babel-loader.
- Set up Postgres database.
- Create the Greeting model with 5 different greeting messages stored.
- Create the API endpoint that selects a random greeting.
- Create the App component with react-router and render it in the root view.
- Create the Greeting component that will display a greeting.
- Set up redux for managing the messages state.

### CODE REVIEW

File: db/migrate/20230221094708_create_messages.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateMessages < ActiveRecord::Migration[7.0]
+  def change
+    create_table :messages do |t|
```
Comment (Davidosky007) [inline db/migrate/20230221094708_create_messages.rb:3]: * Kindly create 5 different greetings as stated in the [project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=create%205%20different%20greetings)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @adel-gu 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230221094708_create_messages.rb (hunk 1 above)
Comment (adel-gu) [reply db/migrate/20230221094708_create_messages.rb:3]: Thank you very much @Davidosky007 for mentioning this point 👏🏼 I'll work on it right away 💪🏼

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @adel-gu👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- Highly organized code. ✔️
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**